src/browser/api/application.js

```diff
@@ -837,18 +861,27 @@ Application.setTrayIcon = function(identity, iconUrl, callback, errorCallback) {
     });
 };
 
-Application.setZoomLevel = function(identity, level) {
-    const app = coreState.appByUuid(identity.uuid);
+Application.setZoomLevel = function(identity, level, callback, errorCallback) {
```

Comment:
https://github.com/HadoukenIO/core/blob/develop/src/browser/api_protocol/api_handlers/application.js#L94 callback and errorcallback aren't even passed in here

Reply:
Please see other comments.